spinn_front_end_common/interface/front_end_common_interface_functions.py

```diff
@@ -530,17 +537,22 @@ def wait_for_execution_to_complete(
 
             processors_not_finished = self._txrx.get_core_state_count(
                 app_id, CPUState.RUNNING)
-            if processors_not_finished > 0:
-                logger.info("Simulation still not finished or failed - "
-                            "waiting a bit longer...")
-                time.sleep(0.5)
+            #if processors_not_finished > 0:
```

Comment:
Why has this warning been commented out? orginally it was there to support end users when a system was seeming to behave unexpectantely, cores not finishing as expected can be an indication of cores being overloaded and thus lagging behind and therefore results in unpredictable results.